Source/WebKit/WebProcess/WebPage/wc/WCUpdateInfo.serialization.in

```diff
@@ -39,4 +39,65 @@ struct WebKit::WCUpdateInfo {
     Vector<WebKit::WCLayerUpdateInfo> changedLayers;
 }
 
+[Nested,OptionSet] enum class WebKit::WCLayerChange : uint8_t {
+    Children,
+    MaskLayer,
+    ReplicaLayer,
+    Geometry,
+    Preserves3D,
+    ContentsVisible,
+    BackfaceVisibility,
+    MasksToBounds,
+    SolidColor,
+    DebugVisuals,
+    RepaintCount,
+    ContentsRect,
+    ContentsClippingRect,
+    Opacity,
+    Background,
+    Transform,
+    ChildrenTransform,
+    Filters,
+    BackdropFilters,
+    PlatformLayer,
+    RemoteFrame
+};
+
+[CustomHeader] struct WebKit::WCLayerUpdateInfo {
+    [Validator='id'] WebCore::PlatformLayerIdentifier id;
```

Comment:
I think we can remove the validator here as it will return std::nullopt if it does not have enough data
```
    auto id = decoder.decode<WebCore::PlatformLayerIdentifier>();
    if (UNLIKELY(!decoder.isValid()))
        return std::nullopt;
```
We are not constraining id to a value so the validator isn't adding value